lib/command/CommandClient.js

```diff
@@ -100,15 +102,22 @@ class CommandClient extends Client {
                             }
                         }
                     }
+                    return result
                 } else {
                     result += `${this.commandOptions.name} - ${this.commandOptions.description}\n`;
                     if(this.commandOptions.owner) {
                         result += `by ${this.commandOptions.owner}\n`;
                     }
                     result += "\n**Commands:**\n";
+                    var mod = 0;
                     for(const label in this.commands) {
                         if(this.commands.hasOwnProperty(label) && this.commands[label] && this.commands[label].permissionCheck(msg) && !this.commands[label].hidden) {
+                            var lastres = result
                             result += `  **${msg.prefix}${label}** - ${this.commands[label].description}\n`;
+                            if (lastres.length <= 2000 && result.length > 2000){
+                                msg.channel.createMessage(lastres)
```

Comment:
be more efficient if you split the whole thing instead of sending and not discarding